3-UsingAPIs/Week1/assignment/ex2-checkDoubleDigits.js

```diff
@@ -12,8 +12,12 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(number) {
+  return new Promise((resolve, reject) => {
+    number > 9 && number < 100
+      ? resolve('This is a double digit number!')
+      : reject(new Error(`Expected a double digit number but got ${number}`));
```

Comment:
You are using the ternary operator here as an `if-then-else` construct here. It would be more readable and instantly recognizable as an `if-then-else` construct if you used it explicitly:

```suggestion
    if (number > 9 && number < 100) {
      resolve('This is a double digit number!');
    } else {
      reject(new Error(`Expected a double digit number but got ${number}`));
    }
```

Reserve the ternary operator when you need a conditional _expression_, e.g:

```js
const fruit = i === 1 ? 'apple' : 'apples';